src/components/timeline/_timeline.scss

```diff
@@ -0,0 +1,97 @@
+/* ==========================================================================
```

Comment:
There are a few "magic numbers" in this file.
`margin-top: 4px;`, `margin-top: -6px;` etc.
Can we refactor to make these values computed from the size variables?

Reply:
As discussed offline, will need @davidhunter08 input